# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Add CMOV to hwcap, for VIA C3 which is i686 class processor,
# DP:		but does not have 686 optional instruction CMOV.
# DP: Author: GOTO Masanori <gotom@debian.org>
# DP: Upstream status: Not submitted
# DP: Status Details: Not submitted
# DP: Date: 2003-02-21
# DP: Updated 2003-10-12 by dan@debian.org for TLS
# DP: Updated 2004-01-02 to fix building with newer patches by jb.
# DP: Updated 2005-01-02 to fix building with newer patches by gotom.

2005-01-02  GOTO Masanori  <gotom@debian.org>

	* sysdeps/i386/dl-procinfo.h: Update to add CMOV for VIA C3 chip.

--- sysdeps/i386/dl-procinfo.h	10 Mar 2004 18:29:31 -0000	1.3
+++ sysdeps/i386/dl-procinfo.h	2 Jan 2005 09:13:55 -0000
@@ -57,7 +57,7 @@
   HWCAP_I386_AMD3D = 1 << 31,
 
   /* XXX Which others to add here?  */
-  HWCAP_IMPORTANT = (HWCAP_I386_XMM2)
+  HWCAP_IMPORTANT = (HWCAP_I386_CMOV | HWCAP_I386_XMM2)
 
 };
